src/commcare_cloud/ansible/roles/commcarehq/templates/localsettings.py.j2

```diff
@@ -723,6 +723,10 @@ WS4REDIS_CONNECTION = {
 }
 {% endif %}
 
+{% if prometheus_celery_pushgateway is defined %}
+PUSHGATEWAY_HOST = '{{ prometheus_celery_pushgateway }}'
```

Comment:
can we change this to `PROMETHEUS_PUSHGATEWAY_HOST`

```suggestion
PROMETHEUS_PUSHGATEWAY_HOST = '{{ prometheus_celery_pushgateway }}'
```

Reply:
is this corresponding name change has been made in `commcare-hq` ?

Reply:
It's in PR: https://github.com/dimagi/commcare-hq/pull/28092